src/main/resources/attackvectors/UnrestrictedFileUploadPayload.properties

```diff
@@ -0,0 +1,38 @@
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved (add case sensitive security), so now to exploit this level you have to create a file with extension .php, and paste there some code, e.g.<br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(1);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  If we go to the URL where the file is stored, the code should be executed. You need to use browser which handle .php files like Use it on Internet Explorer (version OS Build 19042.985).
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_6=By sending a file in proper format hacker can execute code in the user's browser, usually with elevated privileges (due to the fact that many users have an administrator account in Windows and disabled hardened browser policies as default). Typically, a persistent XSS attack is conducted to get sensitive cookie theft and sensitive data theft.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved (only files containing .jpeg and .png extension are accepted), so now to exploit this level you have to a create a file with extension .jpeg.html,and paste there some code, e.g.<br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(1);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  If we go to the URL where the file is stored, the code should be executed.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_7=By sending a file in proper format hacker can execute code in the user's browser, usually with elevated privileges (due to the fact that many users have an administrator account in Windows and disabled hardened browser policies as default). Typically, a persistent XSS attack is conducted to get sensitive cookie theft and sensitive data theft.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved (only files with .jpeg and .png extension are accepted), so now to exploit this level you have to create a file with extension like .html.jpeg, and paste there some code, e.g.<br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(1);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  Now you have to insert a null byte character before .jpeg or .png, then this extension will be ignored and a file will be treated as .html. If we go to the URL where the file is stored, the code should be executed.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_8=By sending a file in proper format hacker can execute code in the user's browser, usually with elevated privileges (due to the fact that many users have an administrator account in Windows and disabled hardened browser policies as default). Typically, a persistent XSS attack is conducted to get sensitive cookie theft and sensitive data theft.<br/>\
```

Comment:
As this payload doesn't trick the user in the execution of code so please modify the payload accordingly.